shanoir-ng-import/src/main/java/org/shanoir/ng/importer/dcm2nii/NIfTIConverter.java

```diff
@@ -148,23 +148,27 @@ public String getDisplayString() {
 	}
 
 	public boolean isDcm2Nii() {
```

Comment:
Except maybe for `isDicomifier` which is used in multiple classes  I don't really see the point of these `is[type]` method, when the converter type is otherwise publicly accessible